packages-next/admin-ui/src/components/Navigation.tsx

```diff
@@ -64,6 +71,21 @@ const NavItem: FunctionComponent<NavItemProps> = ({ href, children }) => {
   );
 };
 
+const SignoutButton: FunctionComponent = () => {
+  const [endSession, { loading, data }] = useMutation(END_SESSION);
+  useEffect(() => {
+    if (data?.endSession) {
+      top.location.reload();
+    }
+  }, [data]);
+
+  return (
+    <Button size="small" isLoading={loading} onPress={() => endSession()}>
```

Comment:
Not about this PR but can we pls rename onPress to onClick? Calling it onPress just makes things confusing. (if it was actually a special thing, sure let's call it onPress but it's not)

Reply:
actually, from seeing #3745, all we need to do is remove the onPress prop.